Sources/DemoKitSnapshot/XCTestCase+Extensions.swift

```diff
@@ -34,6 +34,9 @@ extension XCTestCase {
     public func snapshotTest(
         demoable: any Demoable,
         record: Bool = false,
+        // https://github.com/pointfreeco/swift-snapshot-testing/pull/628#issuecomment-1256363278
+        precision: Float = 0.98,
+        perceptualPrecision: Float = 0.98,
```

Comment:
trying to make sure it runs well with a bit of leeway, if it does, will try to drop it 🙂
perceptualPrecision should be on though. if it works as advertised, it will not hurt and maybe save someone's time down the line

Reply:
seems to work!
https://github.com/finn-no/FinniversKit/pull/1345